packages/block-library/src/classic/edit.js

```diff
@@ -1,3 +1,5 @@
+/* global wp */
```

Comment:
An alternative here is to use `window.wp`.

Reply:
Yes, I discussed it with @aduth at the end of the day. I will update.